ubtue_SAGE_Journals.js

```diff
@@ -0,0 +1,392 @@
+{
+	"translatorID": "4ff2bcd8-968c-49bd-94d2-e99c41aeb842",
+	"label": "SAGE Journals UBTue",
+	"creator": "Sebastian Karcher, Johannes Riedl",
+    //"target": "^https?://journals\\.sagepub\\.com(/doi/((abs|full|pdf)/)?10\\.|/action/doSearch\\?|/toc/)",
+	"target": "^THISWILLNEVERMATCH$)",
```

Comment:
Is this for debugging or because this is the "hidden" translator that should never match? a comment would be nice.